frontend/src/APIClients/mutations/ClassMutations.ts

```diff
@@ -11,4 +11,10 @@ const CREATE_CLASS = gql`
   }
 `;
 
-export default CREATE_CLASS;
+export const CREATE_STUDENT = gql`
+  mutation CreateStudent($student: StudentRequestDTO!, $classId: String!) {
```

Comment:
```suggestion
  mutation CreateStudent($student: StudentRequestDTO!, $classId: ID!) {
```

Reply:
@joyce-shi This messes up the mutation since the resolver is expecting a string, is there a type defined in the backend that I can use because I can't find one within the` ClassService.ts` interface file

Reply:
I just pulled the branch and made the change, it should work as long as you update the `classType` file appropriately. No need to make any type changes in the backend. You can reference `GET_CLASS_BY_TEST_SESSION` in `ClassQueries` as an example.

Reply:
Ahh I see, I have made the change!